### DISCUSSION TITLE
Export postscript not running when using quick export feature.

### DISCUSSION THREAD

**Discussion by markus-utke** [retorquere/zotero-better-bibtex#3080] [Q&A] [answered]
Hello,
I wrote a small postscript that modifies some conference titles when exporting items and it works great. 
I now noticed that the postscript does not run when using the quick export feature, i.e. either ctrl+shift+c or dragging and dropping an item. Is this intended behavior? Is there a way to enable the postscript?
Thanks in advance!

**Comment by retorquere** [comment, marked as answer]
The postscript should always run, if it doesn't for you, that would be a bug, and you'd have to open an issue.

**Reply by markus-utke** [reply]
Thanks for the quick answer! I'll open an issue then.
